Gulpfile.js

```diff
@@ -38,14 +41,25 @@ function compile(advanced = false) {
         .pipe(sourcemaps.init())
         .pipe(closureCompiler(options))
         .pipe(sourcemaps.write('/'))
-        .pipe(gulp.dest('dist'));
+        .pipe(gulp.dest('dist/js'));
```

Comment:
We need to change several other places for this, because they point to dist. I also think it's OK to keep both html and css files directly under dist, no need to separate in two folders.